### PR TITLE
Setting Switch state not visible properly in Dark AMOLED basic theme

### LINKED ISSUE #3017: Switch not visible properly
**Describe the bug**
Switch not visible properly when user  is in Dark AMOLED basic theme

**To Reproduce**
1. Go to  Setting
2. Click on Basic Theme
3. select Dark AMOLED 
3. Go to Accounts
4. See switch not visible properly if  switch is in off state

**Expected behavior**
User should able to see whether switch is off or on properly

**Screenshots**

Screenshot 2020-02-15 at 2 31 20 PM

**Smartphone Info:**

|               | |
|---------------|-|
|Device         | Pixel 2|
|Android Version| 7.1 |

**Would you like to work on the issue?**

- [ ✓ ] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by AjayThakur12**
Fixed #3017 

Changes: 
Added a function in ThemeHelper Class.
Created a function which handle switch colour update on change in Basic theme(in setting section).

Screenshots of the change: 
Screenshot 2020-02-17 at 10 00 43 PM

### CODE REVIEW

File: app/src/main/java/org/fossasia/phimpme/gallery/util/ThemeHelper.java [hunk 1]
```diff
@@ -125,6 +125,22 @@ public int getBackgroundColor() {
...
   }
 
+  public int getUpdatedSwitchColor() {
+    int color;
+    switch (baseTheme) {
+      case DARK_THEME:
+        color = getColor(R.color.md_dark_background);
+        break;
+      case AMOLED_THEME:
+        color = getColor(R.color.md_brown_300);
```
Comment (abishekvashok) [inline app/src/main/java/org/fossasia/phimpme/gallery/util/ThemeHelper.java]: Can you check the material color guide and maybe get a more interactive and interesting color?

Comment (AjayThakur12) [reply app/src/main/java/org/fossasia/phimpme/gallery/util/ThemeHelper.java]: sure @abishekvashok  i will look for better colour in material colour guide

Comment (abishekvashok) [reply app/src/main/java/org/fossasia/phimpme/gallery/util/ThemeHelper.java]: Cool then :D

Comment (AjayThakur12) [reply app/src/main/java/org/fossasia/phimpme/gallery/util/ThemeHelper.java]: @abishekvashok  please review changes

**Pull request merged**